setup.py

```diff
@@ -294,11 +327,77 @@ def adjust_installation_command():
                 else:
                     readme_lines[ln] = "  pip install Pootle\n"
 
+    def replace_urls():
+        from pootle.core.utils import version
+        branch = version.get_git_branch()
+        if branch is not None:
+            branch_escape = branch.replace('/', '%2F')
+        for ln, line in enumerate(readme_lines):
+            for pattern, replace, rewrite_type in (
```

Comment:
Nit: put this tuple of tuples out of the `for` loops to increase code readability.